test/functional/JLM_Tests/src/org/openj9/test/management/TestOperatingSystemMXBean.java

```diff
@@ -63,6 +63,21 @@
 	private static final int SLEEPINTERVAL = 5;
 	private static final int NTHREADS = 1;
 	private static final int NITERATIONS = 10;
+	
+	static boolean isPhysicalMemoryTestExcluded() {
+		/* Exclude on Linux aarch64 and x86-64
+		 * https://github.com/eclipse/openj9/issues/12038
+		 */
+		String osname = System.getProperty("os.name");
+		if (!("Linux").equals(osname)) {
+			return false;
+		}
+		String arch = System.getProperty("os.arch");
+		if (!"amd64".equals(arch) && !"aarch64".equals(arch)) {
+			return false;
+		}
+		return true;
+	}
```

Comment:
This can be private, and I suggest it's easier to follow if the conditions are inverted:
```
	private static boolean isPhysicalMemoryTestExcluded() {
		/* Exclude on Linux aarch64 and x86-64
		 * https://github.com/eclipse/openj9/issues/12038
		 */
		String osname = System.getProperty("os.name");
		if ("Linux".equals(osname)) {
			String arch = System.getProperty("os.arch");
			if ("aarch64".equals(arch) || "amd64".equals(arch)) {
				return true;
			}
		}
		return false;
	}
```